TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2018/3l_semiddbkg/mca-data.txt

```diff
@@ -1,9 +0,0 @@
-### 2018
```

Comment:
We are deleting this again, although it is not deleted in the other years. Can you confirm that this is intentional?